seshdash/static/sesh-ui/js/sesh_dashboard.js

```diff
@@ -165,21 +165,36 @@ get_high_chart( date, HighChartHighPvProduction, HighChartHighCloudCover);
      $.post('/notifications',jsonData, function(data){
 
           var alertData = JSON.parse(data);
-          $('#pop').html(alertData[0].alerts_counter);
+ 
           var out= $("#alert-notification-table");
           var element = '';
           var i;
+ 
+
+
+         for (i=0 ; i<alertData.length ;i++){
```

Comment:
Why are you iterating over the data twice? You're already iterating over alert data on line 186.